src/mocks/constants.ts

```diff
@@ -516,7 +516,7 @@ export const MOCK_SUCCESS_LAUNCHED_SITE_LAUNCH_DTO: SiteLaunchDto = {
 }
 
 export const MOCK_FAILURE_LAUNCHED_SITE_LAUNCH_DTO: SiteLaunchDto = {
-  siteStatus: "FAILURE",
+  siteLaunchStatus: SiteLaunchBEStatus.failed,
```

Comment:
need to capitalise

Reply:
PascalCase for all

Reply:
358863416b34d5c0affb55eecc6ff6c56be1fa99